e2e/cypress/integration/ui/settings/project_mgmt.spec.ts

```diff
@@ -75,171 +77,190 @@ describe('project management', () => {
+    cy.get('app-notification.info').contains(`Created project ${projectID}`);
+    cy.get('app-notification.info chef-icon').click();
+
     cy.contains(projectName).should('exist');
     cy.contains(projectID).should('exist');
 
     cy.url().should('include', '/settings/projects');
   });
 
-  it('addPolicies checkbox is always checked on modal open', () => {
+  it('can create a project with a custom ID without generating associated policies', () => {
     cy.get('[data-cy=create-project]').contains('Create Project').click();
+    cy.get('app-project-list chef-modal').should('have.class', 'visible');
 
-    cy.get('app-create-object-modal chef-checkbox')
-      .should('have.attr', 'aria-checked', 'true');
-
-    cy.get('app-create-object-modal chef-button').contains('Cancel').click();
-  });
-
-  it('can open the new project\'s details page', () => {
-    cy.get('[data-cy=project-details]').contains(projectName).click();
-    cy.url().should('include', `/settings/projects/${projectID}`);
-  });
-
-  // something is occasionally resetting the projects filter to (unassigned)
-  itFlaky('can create a rule for the new project', () => {
-    cy.get('app-project-details').contains('Edits are pending').should('not.exist');
-    cy.get('app-project-details app-authorized button').contains('Create Rule').click();
-
-    cy.url().should('include', `/settings/projects/${projectID}/rules`);
-    cy.get('app-project-rules chef-page').should('be.visible');
-
-    cy.get('#create-name input').focus().type(ruleName, { delay: typeDelay })
-      .should('have.value', ruleName);
-    cy.get('[data-cy=edit-id]').click();
-
-    cy.get('#create-id input').should('have.value', ruleID);
-
-    // Verify correct attributes/operators are selectable for each resource
-    // Event
-    cy.get('#create-type-dropdown').select('Event');
-    cy.get('[data-cy=attribute-dropdown]').select('Chef Organization');
-    cy.get('[data-cy=attribute-dropdown]').select('Chef Server');
-    cy.get('[data-cy=operator-dropdown]').select('equals');
-    cy.get('[data-cy=operator-dropdown]').select('member of');
-
-    // Node
-    cy.get('#create-type-dropdown').select('Node');
-    const nodeAttributes = ['Chef Organization', 'Chef Server', 'Environment',
-      'Chef Role', 'Chef Tag', 'Chef Policy Name', 'Chef Policy Group'];
+    cy.get('[data-cy=create-name]').focus()
+      .type(projectName).should('have.value', projectName);
 
-    nodeAttributes.forEach((att: string) => {
-      cy.get('[data-cy=attribute-dropdown]').select(att);
-    });
+    cy.get('[data-cy=create-id]').should('not.be.visible');
+    cy.get('[data-cy=edit-button]').contains('Edit ID').click();
+    cy.get('[data-cy=id-label]').should('not.be.visible');
 
-    cy.get('[data-cy=operator-dropdown]').select('member of');
-    cy.get('[data-cy=operator-dropdown]').select('equals');
-    cy.get('[data-cy=rule-value] input').type('Chef Policy Group Foo');
+    // don't create associated project policies
```

Comment:
Helpful to provide a bit more context
```suggestion
    // checkbox is selected by default; uncheck so we do NOT create associated project policies
```